include/backend/Scalar.hpp

```diff
@@ -392,6 +392,9 @@ namespace cytnx {
       return nullptr;
     }
 
+    /**
+     * @deprecated This method is not in use anymore.
+     */
     virtual void *get_raw_address() const {
       cytnx_error_msg(true, "[ERROR] Void Type Scalar cannot have operation!!%s", "\n");
       return nullptr;
```

Comment:
Mark this function as deprecated. I may be deleted or renamed to `data()` like `std` convention.